ui/app/AppLayouts/Profile/views/AdvancedView.qml

```diff
@@ -188,26 +186,39 @@ SettingsContentBase {
             StatusSettingsLineButton {
                 anchors.leftMargin: 0
                 anchors.rightMargin: 0
-                text: qsTr("POC Wallet Connect")
-                visible: root.advancedStore.isDebugEnabled
-
+                text: qsTr("Archive Protocol Enabled")
+                isSwitch: true
+                switchChecked: root.advancedStore.archiveProtocolEnabled
                 onClicked: {
-                    Global.popupWalletConnect()
+                    root.advancedStore.toggleArchiveProtocolEnabled()
                 }
             }
-            /////////////////////////////////////////////////////
 
             StatusSettingsLineButton {
                 anchors.leftMargin: 0
                 anchors.rightMargin: 0
-                text: qsTr("Archive Protocol Enabled")
+                text: qsTr("ENS Community Permissions Enabled")
                 isSwitch: true
-                switchChecked: root.advancedStore.archiveProtocolEnabled
+                switchChecked: root.advancedStore.ensCommunityPermissionsEnabled
                 onClicked: {
-                    root.advancedStore.toggleArchiveProtocolEnabled()
+                    root.advancedStore.toggleEnsCommunityPermissionsEnabled()
                 }
             }
 
+            /////////////////////////////////////////////////////
```

Comment:
Is this intentional ?

Reply:
I think it was added by the WalletConnect team. I just moved their setting lower so that the Community settings stay together